# sends a command to the current controller the robot is connected to.
# OpenRAVE sends directly to ControllerBase::SendCmd,
# ControllerBase::SupportsCmd is also used to check for support.
string sessionname
int32 sessionid

uint32 bodyid
string cmd
---
# 1 if command was accepted, 0 if not
uint8 success
